.gitignore

```diff
@@ -1,3 +1,4 @@
+out
```

Comment:
This one is too general of an ignore, feel free to add your personal '~/.gitignore'